app/Models/User.php

```diff
@@ -362,7 +362,7 @@ public static function lookup($username_or_id, $lookup_type = null, $find_all =
                 break;
 
             default:
-                if (is_numeric($username_or_id)) {
+                if (ctype_digit($username_or_id)) {
```

Comment:
`ctype_digit(1)` is false

Reply:
it's a string though

Reply:
but is it (it's called in quite a few places)

Reply:
yeah, every call (with default lookup) is a string. should I cast it just to be safe?

Reply:
yes, before someone gets confused why `lookup(48)` works but returns wrong user.